meal-planner-fredericton-ui/src/dashboard/Dashboard.jsx

```diff
@@ -303,10 +342,18 @@ const Dashboard = props => {
                     </TableCell>
                     {/* <TableCell align="right"><Link to='/copy-plan'><FileCopyIcon /></Link></TableCell> */}
                     <TableCell align="right">
-                      <Link onClick={async () => {
-                        await doDeleteMeapPlan(row);
+                      {/* <Link onClick={
+                        async () => {
+                        await doDeleteMealPlan(row);
                         props.relay.refetch();
-                      }}>
+                      }}> */}
+                      <Link onClick={() => {
+                        setCurrentRow(row);
+                        console.log('set current row mealplan', row);
+                        setShowDelete(true);
+                        console.log('delete dialog is set to open', showDelete);
+                      }
+                        }>
```

Comment:
This sets the current row that is selected and sets the condition to render the delete dialog.